### ISSUE TITLE
Can you provide some documentation?

### ISSUE THREAD

**Issue by Monikasultania** [Kaljurand/speechutils#8]
Please provide some documentation to use it.

**Comment by Kaljurand** [comment]
Yes, the documentation is currently missing. What parts would interest you the most? I can try to focus on them first then.

As a usage example, you could also look into https://github.com/Kaljurand/K6nele
